docs/examples/use_cases/tensorflow/efficientdet/train.py

```diff
@@ -0,0 +1,232 @@
+            callbacks.append(
+                tf.keras.callbacks.ModelCheckpoint(
+                    filepath=os.path.join(
+                        ckpt_dir, "".join([args.model_name, ".{epoch:02d}.h5"])
+                    ),
+                    save_weights_only=True,
+                )
+            )
+
+        if args.log_dir:
+            log_dir = args.log_dir
+            if not tf.io.gfile.exists(log_dir):
+                tf.io.gfile.makedirs(log_dir)
+            callbacks.append(
+                tf.keras.callbacks.TensorBoard(log_dir=log_dir, update_freq="epoch")
+            )
+
+        model.fit(
+            train_dataset,
+            epochs=args.epochs,
+            steps_per_epoch=args.train_steps,
+            initial_epoch=initial_epoch,
+            callbacks=callbacks,
+            validation_data=eval_dataset if args.eval_during_training else None,
+            validation_steps=args.eval_steps,
+            validation_freq=args.eval_freq,
+        )
+
+        if args.eval_after_training:
+            print("Evaluation after training:")
+            model.evaluate(eval_dataset, steps=args.eval_steps)
+
+        model.save_weights(args.output)
+
+
+if __name__ == "__main__":
+    import argparse
+    from argparse_utils import enum_action
+
+    parser = argparse.ArgumentParser()
+    parser.add_argument(
+        "--initial_epoch",
+        type=int,
+        default=0,
+        help="epoch from which to start training",
+    )
+    parser.add_argument(
+        "--epochs", type=int, default=300, help="epoch on which training should finish"
+    )
+    parser.add_argument(
+        "--input",
+        action=enum_action(utils.InputType),
+        required=True,
+        help="input type",
+    )
+    parser.add_argument(
+        "--images_path",
+        required=False,
+        help="Path to COCO images",
+    )
+    parser.add_argument(
+        "--annotations_path",
+        required=False,
+        help="Path to COCO annotations",
+    )
+    parser.add_argument(
+        "--train_file_pattern",
+        required=False,
+        help="glob pattern for TFrecord files with training data",
+    )
+    parser.add_argument("--train_batch_size", type=int, default=64)
+    parser.add_argument(
+        "--train_steps", type=int, default=2000, help="number of steps in each epoch"
+    )
+    parser.add_argument(
+        "--eval_file_pattern",
+        help="glob pattern for TFrecord files with evaluation data, "
+        "defaults to TRAIN_FILE_PATTERN if not given",
+    )
+    parser.add_argument(
+        "--eval_steps", type=int, default=5000, help="number of examples to evaluate"
+    )
+    parser.add_argument(
+        "--eval_freq", type=int, default=1, help="during training evalutaion frequency"
+    )
+    parser.add_argument(
+        "--eval_during_training",
+        action="store_true",
+        help="whether to run evaluation every EVAL_FREQ epochs",
```

Comment:
I believe this formatting is how we should refer to other arguments
```python
"whether to run evaluation every `eval_freq` epochs"
```

Reply:
Fixed.